ambry-api/src/main/java/com/github/ambry/config/RouterConfig.java

```diff
@@ -253,13 +257,41 @@ public class RouterConfig {
   @Default("3")
   public final int routerPutRequestParallelism;
 
+  /**
+   * Paranoid durability: Max parallel requests issued at a time in the local data center for a chunk.
+   */
+  @Config(ROUTER_PUT_LOCAL_REQUEST_PARALLELISM)
+  @Default("3")
+  public final int routerPutLocalRequestParallelism;
+
+  /**
+   * Paranoid durability: Max parallel requests issued at a time in remote data centers for a chunk.
+   */
+  @Config(ROUTER_PUT_REMOTE_REQUEST_PARALLELISM)
+  @Default("0")
```

Comment:
default value should be at least 1. I think we should use 1 as default value.

Reply:
Agreed, and changed this as requested.